README.md

```diff
@@ -37,7 +37,7 @@ This fork is built on top of [commit 4df6f97](https://github.com/google/boringss
 
 liboqs and this integration are provided "as is", without warranty of any kind.  See the [LICENSE](https://github.com/open-quantum-safe/liboqs/blob/main/LICENSE.txt) for the full disclaimer.
 
-**N.B.: WE DO NOT REGULARLY TEST INTEROPERABILITY WITH [OQS-OPENSSL](https://github.com/open-quantum-safe/openssl), INSTEAD ONLY DOING SO AS AND WHEN THE NEED MIGHT ARISE. AS A CONSEQUENCE, THIS PROJECT, AS WELL AS THE CHROMIUM DEMO THAT IS BUILT ON TOP OF IT, MAY NOT INTEROPERATE WITH OTHER COMPONENTS IN THE OQS INFRASTRUCTURE SUCH AS THE [OQS PROVIDER](https://github.com/open-quantum-safe/oqs-provider) AND THE [OQS TEST SERVER](http://test.openquantumsafe.org/).**
+**N.B.: THIS PROJECT, AS WELL AS THE CHROMIUM DEMO THAT IS BUILT ON TOP OF IT, MAY NOT INTEROPERATE WITH OTHER COMPONENTS IN THE OQS INFRASTRUCTURE SUCH AS THE [OQS PROVIDER](https://github.com/open-quantum-safe/oqs-provider) AND THE [OQS TEST SERVER](http://test.openquantumsafe.org/).**
```

Comment:
Does this mean we do not even strive to maintain interoperability between BoringSSL and OpenSSL any more, e.g., trying to use the same code points for the same algs?

Reply:
They interoperate very well. That's why we use "MAY"

Reply:
Good to hear. What about adding a CI job confirming this then?

Reply:
Will try to do this later. Sometimes algorithms used by the test server aren’t the latest, so CI won’t pass.